.github/workflows/spark-tests.yml

```diff
@@ -5,6 +5,13 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ '*' ]
+    paths:
+      - 'listenbrainz_spark/**'
+      - 'requirements_spark.txt'
+      - 'requirements_development.txt'
+      - 'Dockerfile.spark'
+      - 'docker/docker-compose.spark*.yml'
+      - 'test.sh'
```

Comment:
Spark code is also affected by the request consumer but currently we do not have any integrated testing so I haven't included that in the paths.